JSTests/test262/expectations.yaml

```diff
@@ -1,6 +1,48 @@
 ---
 test/annexB/language/function-code/block-decl-func-skip-arguments.js:
   default: 'Test262Error: Expected SameValue(«"function arguments() {}"», «"[object Arguments]"») to be true'
+test/built-ins/ArrayBuffer/prototype/resize/this-is-immutable-arraybuffer-object.js:
```

Comment:
I think we should exclude `immutable-array-buffer` because of we don't implement it yet

Reply:
Updated

Reply:
Because `JSTests/test262/test/built-ins/ArrayBuffer/prototype/slice/species-returns-immutable-arraybuffer.js` should be included.